File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    big = max(sizes[0])
    small = min(sizes[0])
    
    for a,b in sizes:
        if a>b:
            if a > big:
                big = a
            if b > small:
                small = b
        else:
            if b > big:
                big = b
            if a > small:
                small = a
    return small*big

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(vector<vector<int>> sizes) {
    int answer = 0;
    int h = 0;
    int w = 0;
    int n1, n2;
    for(int i = 0; i < sizes.size(); i++){
        n1 = sizes[i][0];
        n2 = sizes[i][1];
        if(n1>n2) swap(n1,n2);
        if(h < n1) h = n1;
        if(w < n2) w = n2;
    }
    return w*h;
}